db/cache/charts.go

```diff
@@ -468,14 +474,18 @@ func (charts *ChartData) ReorgHandler(wg *sync.WaitGroup, c chan *txhelpers.Reor
 			commonAncestorHeight := uint64(data.NewChainHeight) - uint64(len(data.NewChain))
 			charts.mtx.Lock()
 			defer charts.mtx.Unlock()
+			newHeight := int(commonAncestorHeight) + 1
+			log.Debug("ChartData.ReorgHandler snipping blocks height to %d", newHeight)
 			charts.Blocks.Snip(int(commonAncestorHeight) + 1)
```

Comment:
Did you want to feed `newHeight` to `Snip`?

Reply:
I need some coffee.